doc/howtos/booting-and-installation.rst

```diff
@@ -127,6 +127,73 @@ includes a checksum verification, you can also use the traditional :command:`dd`
 Unplug the removable media from your development system and you're ready to plug
 it into your target system.
 
+Installing to internal media on generic x86 EFI platforms
+=========================================================
+
+This section applies to those cases where the device is x86 compatible (ex: Gigabyte NUC).
+The simpler way is to follow these steps:
+
+#. Create a Bootable Media using both an image that is compatible with the device
+   and a supported removable media (SD card or USB stick).
+#. Boot the device with the removable media created at the previous step.
+#. Once the device is booted, identify the booted media, for example by executing::
+
+      # mount
+
+   and locating in its output which partition is mounted ``on /``.
+   This will be a partition belonging to the source block device.
+   Example::
+
+      /dev/sdb3 on / type ext4 (rw,realtime,i_version,data=ordered)
+
+   Shows that the root partition is on ``/dev/sdb3``, therefore the source block device will
+   be ``/dev/sdb``.
+#. Identify the block device representing the internal storage.
+   This part is really specific to each platfrom: typically the internal storage is
+   associated to ``/dev/sda`` for hard disks and SSD units and to ``/dev/mmcblk0`` for eMMC
+   devices, but it should be confirmed against the board manual or the BIOS, if it provides
+   such type of information.  Lacking that, one can run::
+
+      # lsblk
```

Comment:
sorry to be late for this review and whole discussion, but the `lsblk` allows ways to only show the actual device, not each partition, as well as limit the block type (to be only disks, etc). Last but not least one can show some properties (ie: hotpluggable, mount path...) and filter with grep. I've shared the command lines many times, but now I can't try one due lack of dev machine :-)